apis_core/core/templates/partials/footer-left.html

```diff
@@ -17,8 +17,6 @@
 {% git_repository_url as repository_url %}
 {% if repository_url %}
   <a href="{{ repository_url }}" title="Git repository">
```

Comment:
```suggestion
  <a href="{{ repository_url }}" title="Code repository">
```
Given that we exchange the Git icon for a more general Code icon I suggest we also change the text accordingly.

Reply:
I'm thinking just "source code" might make even more sense :thinking:, but I updated it to "Code repository" for now.